app/backend/src/couchers/models.py

```diff
@@ -1475,6 +1475,8 @@ class BackgroundJobType(enum.Enum):
     add_users_to_email_list = 6
     # payload: google.protobuf.Empty
     send_request_notifications = 7
+    # payload: google.protobuf.Empty
+    enforce_community_membership = 7
```

Comment:
Repeated enum value. Please change all of these to `enum.auto()` as these are categorical, not ordinal. (I think they were left as non-auto due to some merge timing between the PR that added bg jobs and where we switched to auto)

Reply:
done